app/Resources/views/programs/index.html.twig

```diff
@@ -19,13 +19,13 @@
             <thead>
                 <tr>
                     <th>{{ msg('program') }}</th>
-                    <th>{{ msg('events') }}</th>
-                    <th>{{ msg('participants') }}</th>
-                    <th>{{ msg('new-editors') }}</th>
-                    <th>{{ msg('day-retention', [30]) }}</th>
-                    <th>{{ msg('pages-created') }}</th>
-                    <th>{{ msg('pages-improved') }}</th>
                     <th></th>
+                    <th class="center-align-cell">{{ msg('events') }}</th>
+                    <th class="center-align-cell">{{ msg('participants') }}</th>
+                    <th class="center-align-cell">{{ msg('new-editors') }}</th>
+                    <th class="center-align-cell">{{ msg('day-retention', [30]) }}</th>
+                    <th class="center-align-cell">{{ msg('pages-created') }}</th>
+                    <th class="center-align-cell">{{ msg('pages-improved') }}</th>
```

Comment:
Since we're only using `center-align-cell` to center text, we might could use the Boostrap `text-center` class instead of creating a new rule.

Reply:
Done.